Barotrauma/BarotraumaClient/WindowsClient.csproj

```diff
@@ -88,6 +2239,7 @@
     <None Remove="libopenal.1.dylib" />
     <None Remove="libSDL2-2.0.0.dylib" />
     <None Remove="libwebm_mem_playback_x64.dylib" />
+    <None Remove="SharedSource\LuaCs\Plugins\**" />
```

Comment:
What's this for?

Reply:
I've reverted the WindowsClient.csproj, WindowsServer.csproj and WindowsSolution.sln